.github/workflows/ci.yml

```diff
@@ -226,9 +226,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-# this occupies 8084 port on Ubuntu 20.04 and we don't use it.
-      - name: Remove mono blocking 8084 port
-        run: sudo kill -9 $(sudo lsof -t -i:8084)
+        # This occupies port 8084 unnecessarily
+      - name: Stop Mono
+        run: |
+          sudo systemctl stop mono-xsp4.service
+          sudo systemctl disable mono-xsp4.service
```

Comment:
I think I tried this in the past, but let's wait on the pipeline result.

Reply:
OK lets see. But that would be a bug in their stop script..

Reply:
That looks about right

```
sudo systemctl stop mono-xsp4.service
  sudo systemctl disable mono-xsp4.service
  shell: /usr/bin/bash -e {0}
  env:
    IMG: local-tomcat
    HTTPD_IMG: local-httpd
    DEBUG: on
    FOREVER_PAUSE: 100
    ITERATION_COUNT: 2
    TOMCAT_CYCLE_COUNT: 2
mono-xsp[4](https://github.com/modcluster/mod_proxy_cluster/actions/runs/9989603001/job/27608578477?pr=256#step:3:4).service is not a native service, redirecting to systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable mono-xsp4
```

Reply:
```
Running JBCS-1236                                  ... NOK
```

I'm afraid that it does not work. There is a problem with tomcat5 (8080 + offset 4).

Reply:
```
18-Jul-2024 10:47:44.081 SEVERE [main] org.apache.catalina.util.LifecycleBase.handleSubClassException Failed to initialize component [Connector["http-nio-127.0.0.5-8084"]]
	org.apache.catalina.LifecycleException: Protocol handler initialization failed
		at org.apache.catalina.connector.Connector.initInternal(Connector.java:1030)
		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:122)
		at org.apache.catalina.core.StandardService.initInternal(StandardService.java:525)
		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:122)
		at org.apache.catalina.core.StandardServer.initInternal(StandardServer.java:961)
		at org.apache.catalina.util.LifecycleBase.init(LifecycleBase.java:122)
		at org.apache.catalina.startup.Catalina.load(Catalina.java:708)
		at org.apache.catalina.startup.Catalina.load(Catalina.java:731)
		at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
		at java.base/java.lang.reflect.Method.invoke(Method.java:580)
		at org.apache.catalina.startup.Bootstrap.load(Bootstrap.java:302)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:472)
	Caused by: java.net.BindException: Address already in use
		at java.base/sun.nio.ch.Net.bind0(Native Method)
		at java.base/sun.nio.ch.Net.bind(Net.java:565)
		at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:344)
		at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:301)
		at org.apache.tomcat.util.net.NioEndpoint.initServerSocket(NioEndpoint.java:239)
		at org.apache.tomcat.util.net.NioEndpoint.bind(NioEndpoint.java:194)
		at org.apache.tomcat.util.net.AbstractEndpoint.bindWithCleanup(AbstractEndpoint.java:1304)
		at org.apache.tomcat.util.net.AbstractEndpoint.init(AbstractEndpoint.java:1317)
		at org.apache.coyote.AbstractProtocol.init(AbstractProtocol.java:632)
		at org.apache.coyote.http11.AbstractHttp11Protocol.init(AbstractHttp11Protocol.java:77)
		at org.apache.catalina.connector.Connector.initInternal(Connector.java:1028)
		... 11 more
```

Reply:
So silly... fixed.